internal/expr/query.go

```diff
@@ -8,36 +8,36 @@ import (
 	"strings"
 )
 
-func (qe *QueryExpr) QueryArgs() []any {
-	return qe.args
+// Params returns the query parameters to pass with the SQL to a database.
+func (qv *QueryValues) Params() []any {
+	return qv.params
 }
 
-func (qe *QueryExpr) HasOutputs() bool {
-	return len(qe.outputs) > 0
+// HasOutputs returns true if the SQLair query contains at least one output
+// expression.
+func (qv *QueryValues) HasOutputs() bool {
+	return len(qv.outputs) > 0
 }
 
-type QueryExpr struct {
+// QueryValues contains all concrete values needed to run a SQLair query on a
+// database.
+type QueryValues struct {
 	sql     string
-	args    []any
+	params  []any
```

Comment:
We have probably talked about this already, but why is this named parameters if the types contained are named `NamedArg` implying arguments (we might have to change the name in the reference spec if we change it here)?

Reply:
In the spec we are using "Input arguments" to refer to the structs and maps from the user, and "query parameters" to refer to what is passed to the database. I've changed this name to try and disambiguate the two in the code.

I've added a comment here (and one for `outputs` too)